features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -191,11 +192,12 @@ export function SidebarNavigation() {
           </LinkList>
 
           <List>
-            <MenuItemButton
+            <MenuItemLink
               text="Support"
               iconSrc="/icons/support.svg"
+              href={supportEmail}
+              isActive={router.pathname === supportEmail}
```

Comment:
Nice job with this implementation. Also with adjusting the Cypress test 😄 

Can the `isActive` prop be truthy here? I mean can `router.pathname` equal `[supportEmail](mailto:support@prolog-app.com?subject=Support Request:)`. If not I'd simply remove this flag or set it to false.